docs/source/datasets.rst

```diff
@@ -22,6 +22,7 @@ Here are all datasets that are available through docTR:
 .. autoclass:: IIIT5K
 .. autoclass:: SVT
 .. autoclass:: SynthText
+.. autoclass:: ICDAR2003
```

Comment:
almost all refs in research papers refer to this dataset as "IC03", would you mind updating this accordingly? :pray: